shortapi/models.py

```diff
@@ -1,3 +1,28 @@
+from django.conf import settings
 from django.db import models
+from user.models import User
+from random import choices
+from string import ascii_letters
+
+
+
+class Link(models.Model):
+    user = models.ForeignKey(User,related_name="user",on_delete=models.CASCADE)
+    original_url = models.URLField()
+    shorten_url = models.URLField(blank=True, null=True)
+    created_at = models.DateTimeField(auto_now_add=True)
+
```

Comment:
updated_at may be a good addition

Reply:
why not?

Reply:
this was a mistake sir already added the update_at

Reply:
this was a mistake sir already added the update_at

Reply:
I don't see it